newspaper/text.py

```diff
@@ -156,3 +156,65 @@ def get_stopword_count(self, content):
         ws.set_stopword_count(len(overlapping_stopwords))
         ws.set_stop_words(overlapping_stopwords)
         return ws
+
+
+class StopWordsJapanese(StopWords):
+    """Japanese segmentation
+    """
+    def __init__(self, language='ja'):
+        super(StopWordsJapanese, self).__init__(language='ja')
+
+    def candidate_words(self, stripped_input):
+        words = []
+        try:
+            import MeCab
+            import os, subprocess
```

Comment:
Can we have a separate try/catch for these two import statements and print a pretty "you must install mecab .. etc" if it is not installed?